### PR TITLE
Hide tips for students, only send events for teachers, and remove placeholder text

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This PR groups a few small fixes:
- hides tips for students (https://codedotorg.atlassian.net/browse/AITT-264)
- removes placeholder tooltips (https://codedotorg.atlassian.net/browse/AITT-261)
- stops sending an event for students (no Jira task)

Screenshot of the AiAssessment component without tooltip:
Screenshot 2023-10-20 at 3 41 21 PM

### CODE REVIEW

File: apps/src/templates/rubrics/AiAssessmentBox.jsx
Comment (kobryan0619) [inline apps/src/templates/rubrics/AiAssessmentBox.jsx:61]: It seems like since we have integrated the confidence scores, we SHOULD have this tool tip here.  Obviously I will defer to Tess/Molly, but as a teacher, I would be frustrated if AI has a "low confidence" but I don't actually know what that means or what I should do about that vs a "high confidence".   That information isn't really useful without context.   If we aren't able to come up with text that provides that context I might argue that we shouldn't be providing confidence scores to teachers.  Also, it is possible that you and Tess already had this conversation and landed on removing this text, so again, I defer to you two.

File: apps/src/templates/rubrics/LearningGoal.jsx
Comment (kobryan0619) [inline apps/src/templates/rubrics/LearningGoal.jsx:290]: I am glad we added this!

**Review by kobryan0619 — COMMENTED** [review]
I had a bigger question about removing tips here before approving, but once that is resolved, it looks good to me.

File: apps/src/templates/rubrics/AiAssessmentBox.jsx
Comment (bethanyaconnor) [reply apps/src/templates/rubrics/AiAssessmentBox.jsx:61]: Good question! I'm going to move the question over to Slack

**Comment by bethanyaconnor** [comment]
Updated screenshot!

Screenshot 2023-10-23 at 1 56 32 PM

**Review by kobryan0619 — APPROVED** [review]

**Pull request merged**
